src/api/index.tsx

```diff
@@ -142,6 +142,11 @@ type GetResource<SuccessfulResponse> = {
   successfulResponse: SuccessfulResponse;
 };
 
+type EmptyRequestAndResponse = {
+  requestData: undefined;
+  successfulResponse: string;
```

Comment:
If this represents an empty request and response, why is `successfulResponse` declared as a `string`?

Reply:
Because the successful response is an empty string. I guess maybe it could be a string literal? I could try that.

Reply:
Yep, that works ✨